src/js/components/Chart/doc.js

```diff
@@ -97,6 +99,7 @@ export const doc = Chart => {
             'medium',
             'large',
             'xlarge',
+            'fill',
             'full',
```

Comment:
Wondering if we should remove `full` from docs so it won't be confusing to the users about the difference between `fill` and `full` and which one will be better to pick, or maybe at least mention that `full` will be deprecated on the description.

Reply:
I see it has been addressed.